scorecard/scorecard/scorecard.py

```diff
@@ -418,6 +418,7 @@ async def on_startup(app):
 async def on_shutdown(app):
     try:
         await app['asana_client'].close()
+        await app['gh_session'].close()
```

Comment:
we've been nesting these try-finallies to ensure everything gets closed, let's be consistent, though I admit these deeply nested try-finallies seem odd to me.